ChangeLog.d/add-rsa-pss-rsae-support-for-tls12.txt

```diff
@@ -1,8 +1,8 @@
 Features
-   * When GnuTLS/Openssl server is configured in TLS 1.2 mode with a certificate
-     declaring an RSA public key and Mbed TLS is configured in hybrid mode, if
-     `rsa_pss_rsae_*` algorithms are before `rsa_pkcs1_*` ones in this list then
-     the GnuTLS/Openssl server chooses an `rsa_pss_rsae_*` signature algorithm
-     for its signature in the key exchange message. As Mbed TLS 1.2 does not
-     support them, the handshake fails. Add `rsa_pss_rsae_*` support for TLS 1.2
-     to resolve the compitablity issue.
+   * Support rsa_pss_rsae_* signature algorithms in TLS 1.2.
+Bugfix
```

Comment:
```suggestion
   * Support rsa_pss_rsae_* signature algorithms in TLS 1.2.
     Bugfix
```

Reply:
I don't understand this suggested change. “Bugfix” is the title of a section, it needs to be at the start of the line.

Reply:
My bad, sorry
```suggestion
   * Support rsa_pss_rsae_* signature algorithms in TLS 1.2.
   
Bugfix
```
Although I am not sure if this makes any difference to the changelog assembly script.

Reply:
The script doesn't care about blank lines. Any line that starts with a letter is a section header.